### DISCUSSION TITLE
2.4.6

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#123] [Announcements]
- Fixed Providers(Multistream,MoviesDrive,MultiMoives,NetflixMirror)
- Next episode button
- Search history
- Subtitle download support
- Quick search title
- Improved Search UI
- Bugs fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/2.4.6'>2.4.6</a>.</em>

**Comment by yushakoe1** [comment]
By any chance will there be android TV support in the future?

**Reply by Zenda-Cross** [reply]
I tried but didn't work out

**Comment by FearlessPs** [comment]
Video not playing in andriod tv

**Reply by Zenda-Cross** [reply]
app don't support tv
